core/templates/pages/story-viewer-page/story-viewer-page.mainpage.html

```diff
@@ -274,6 +274,25 @@
           width: 100vw;
           z-index: -1;
         }
+    }
+      @media (max-width: 768px) {
+      .navbar-brand {
+        display: none;
+      }
+
+      .d-none {
```

Comment:
Can we conditionally remove this class in the HTML? e.g. using ng-class

Reply:
Or remove d-none and use media-queries to set / unset display value?

Reply:
@kevintab95  d-none is an already used class in the base-components(navbar-breadcrumb) which sets the display value to none as we shift to mobile view using !important. I tried using media queries to set display value to flex in the story-viewer however it always gets over ridden by d-none. Also I am unable to remove it using 'ngclass' as it is applied as a 'class' already.

Reply:
Hm, can you please point me to the d-none you want to override?

Reply:
yeah it is in top-navigation-bar.directive.html (line number 9).

Reply:
I have changed it to .oppia-navbar-brand-name.d-none in the story viewer page. we want the logo to show in the story page and it gets overridden unless !important is used.